ReactWindows/ReactNative.Shared/Views/View/ReactViewManager.cs

```diff
@@ -94,17 +94,52 @@ protected override BorderedCanvas CreateViewInstance(ThemedReactContext reactCon
 
         /// <summary>
         /// Sets whether or not the view is an accessibility element.
+        /// Since "accessible" is an Android-only ReactNative property and also is not in ReactXP at all,
+        /// this method is noop and exists just to avoid crashes if for some reason the property
+        /// gets to this level. For similar functionality see <see cref="SetImportantForAccessibility"/>
```

Comment:
We shouldn't reference ReactXP.

This is a tricky one because other React Native platforms implement `SetAccessible` but we've removed `SetAccessible` from our RN fork in favor of an alternative API. We haven't contributed it back yet because our API still needs work. Rather than noting this stuff in the code, maybe leave the TODO and the comment about it being a no-op and put some notes about why we probably shouldn't implement `SetAccessible` in #557.